utils/R/read.output.R

```diff
@@ -123,8 +123,6 @@ read.output <- function(runid, outdir, start.year=NA,
 
   if(!nofiles){
     for(ncfile in ncfiles) {
-      valid.nc <- tryl(nc_open(ncfile))
```

Comment:
Only problem is that the file might not exist or not be a valid nc file. Maybe one thing to do is wrap the nc_open with a try() block.